### ISSUE TITLE
Negative numbers catched by whitespace_011 for math operator spacing

### ISSUE THREAD

**Issue by stroebjo** [jeremiah-c-leary/vhdl-style-guide#283]
**Environment**
VHDL Style Guide (VSG) version 0.42

**Describe the bug**
Negative numbers, like `-32768`, are catched by rule `whitespace_011` wich enforces spacing between math operates, so the number is fixed to `- 32768`. Since it is a negative number I think in this case there should be no space enforced.

But I don't know if this context can always be distinguished in the whitespace rules?

**To Reproduce**

Example code: 

```

architecture BEH of TEST is

begin

  TEST : process (test) is
  begin

    for i in -32768 to 32767 loop
      -- do stuff
    end loop;

  end process TEST;

end architecture BEH;

```

**Expected behavior**
Should be valid and not be fixed to `for i in - 32768 to 32767 loop`

**Comment by jeremiah-c-leary** [comment]
That is an interesting situation, and like you said: context matters.  I will have to think of a way to classify negative numbers.
